frontend/micro-ui/web/micro-ui-internals/packages/modules/pt/src/pages/Response.js

```
@@ -97,13 +243,13 @@ const Response = (props) => {
     const { Properties = [] } = mutation.data || successData;
     const Property = (Properties && Properties[0]) || {};
     const tenantInfo = tenants.find((tenant) => tenant.code === Property.tenantId);
-    
+
     let tenantId = Property.tenantId || tenantId;
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)